### ISSUE TITLE
can't determine type for tag '<macro name="m3_comp_assist_chip_container_shape">?attr/shapeappearancecornersmall</macro>'

### ISSUE THREAD

**Issue by bilalfawaz** [payfort/fort-android-sdk#33]
When using SDK and creating a new build returns this error 
can't determine type for tag '<macro name="m3_comp_assist_chip_container_shape">?attr/shapeappearancecornersmall</macro>'

please any support?

**Comment by AbhiHearty** [comment]
Yes. I am also facing the same almost spent a lot of time with this and hardly found a solution.

I am using payfort SDK

Can't determine type for tag '<macro name="m3_comp_assist_chip_container_shape">?attr/shapeAppearanceCornerSmall</macro>'

**Comment by r-elbouhi-ibtikar** [comment]
Yes. I am also facing the same issue.
I solved this by downgrade to previous version:

Then change: 

```
implementation("com.amazon:fortpayment:+:release@aar"){
        transitive = true
}
```

To:

```
implementation("com.amazon:fortpayment:2.1.0:release@aar"){
        transitive = true
}
```
